core/templates/simple_layout.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
```

Comment:
Agreed--let's see how different we actually need these, and then we might consider merging them into one file in the future.